frontend/.env

```diff
@@ -0,0 +1 @@
+VITE_PORT=3000
```

Comment:
?

Reply:
My bad. I have reset the head and pushed without the .env